src/main/java/htsjdk/samtools/util/EdgeReadIterator.java

```diff
@@ -98,13 +98,14 @@ void accumulateSamRecord(SAMRecord rec) {
             // 0-based offset from the aligned position of the first base in the read to the aligned position
             // of the current base.
             final int refOffset = refPos - rec.getAlignmentStart();
-            final int refOffsetEnd = refPos - rec.getAlignmentStart() + alignmentBlock.getLength();
+            final int refOffsetEnd = refOffset + alignmentBlock.getLength();
 
+            final int accumulatorStartAdjust = accumulator.isEmpty() ? 0 : rec.getAlignmentStart() - accumulator.get(0).getPosition();
```

Comment:
Is it possible for `accumulator` to be empty? There is code below (lines 145 and 148) where `accumulator.get()` is called without checking `isEmpty()`.

Reply:
I think that this is the only place in the code where it could be empty....